charts/fluentd/templates/configmap.yaml

```diff
@@ -1,133 +0,0 @@
-apiVersion: v1
```

Comment:
We have changed this file to improve the logging, those changes will be gone with this update

Reply:
Yes, that makes sense. However, the kube-gelf image hasn't been updated in a substantial way in 2 years now, which worries me.

Reply:
Could we use https://github.com/fluent/helm-charts/tree/main/charts/fluent-bit instead?

Reply:
I think for Helm charts being regularly updated doesn't matter much, especially that they're small and easily maintainable by us.

Reply:
Fluentd and fluent-bit are a bit different as much as remember and I don't know if fluent-bit can send log message in GELF format. Right now with use `roffe/kube-gelf` image which adds GELF support to fluentd
https://github.com/roffe/kube-gelf/blob/master/Dockerfile